corehq/toggles/__init__.py

```diff
@@ -709,14 +709,24 @@ def _ensure_valid_randomness(randomness):
 
 CASE_LIST_TILE_CUSTOM = StaticToggle(
     'case_list_tile_custom',
-    'USH: Configure custom case list tile',
+    'USH: Configure custom case tile',
     TAG_CUSTOM,
     [NAMESPACE_DOMAIN],
     help_link='https://confluence.dimagi.com/pages/viewpage.action?'
               'spaceKey=saas&title=Allow+Configuration+of+Case+List+Tiles',
     parent_toggles=[CASE_LIST_TILE],
 )
 
+CASE_DETAIL_TILE = StaticToggle(
```

Comment:
could that not be part of CASE_LIST_TILE_CUSTOM? If only USH uses it I don't really see the point in adding a new FF.

Reply:
700b0afd4ea